clean up use of word: easy (#26352)
kubernetes/website: content/en/docs/tasks/administer-cluster/limit-storage-consumption.md

<!-- overview -->

> This example demonstrates an straightforward way to limit the amount of storage consumed in a namespace.

Reviewer: ```suggestion
This example demonstrates how to limit the amount of storage consumed in a namespace.
```
